include/aikido/planner/ompl/Planner.hpp

```diff
@@ -296,17 +298,23 @@ trajectory::InterpolatedPtr planOMPL(
 /// to a state on original path
 /// \param _originalTraj The untimed trajectory obtained from the planner, 
 /// needs simplifying.
-trajectory::InterpolatedPtr simplifyOMPL(statespace::StateSpacePtr _stateSpace,
-                                           statespace::InterpolatorPtr _interpolator,
-                                           distance::DistanceMetricPtr _dmetric,
-                                           constraint::SampleablePtr _sampler,
-                                           constraint::TestablePtr _validityConstraint,
-                                           constraint::TestablePtr _boundsConstraint,
-                                           constraint::ProjectablePtr _boundsProjector,
-                                           double _maxDistanceBtwValidityChecks,
-                                           double _timeout, size_t _maxEmptySteps,
-                                           double _rangeRatio, double _snapToVertex,
-                                           trajectory::InterpolatedPtr _originalTraj);
+std::pair <std::unique_ptr<trajectory::Interpolated>, bool> simplifyOMPL(statespace::StateSpacePtr _stateSpace,
+                                                        statespace::InterpolatorPtr _interpolator,
+                                                        distance::DistanceMetricPtr _dmetric,
+                                                        constraint::SampleablePtr _sampler,
+                                                        constraint::TestablePtr _validityConstraint,
+                                                        constraint::TestablePtr _boundsConstraint,
+                                                        constraint::ProjectablePtr _boundsProjector,
+                                                        double _maxDistanceBtwValidityChecks,
+                                                        double _timeout, size_t _maxEmptySteps,
+                                                        trajectory::InterpolatedPtr _originalTraj);
+
+
+namespace{
+    std::unique_ptr<trajectory::Interpolated> aikido2ompl(::ompl::geometric::PathGeometric *path, 
+                                                    statespace::StateSpacePtr _stateSpace,
+                                                    statespace::InterpolatorPtr _interpolator);
+} // namespace
```

Comment:
Oops. Did not realize I left this here. Was just playing around with the helper function. Didn't want to commit that yet. Shall make the changes suggested :)